core/src/main/java/io/aiven/klaw/controller/TemplateMapController.java

```diff
@@ -376,6 +376,15 @@ public String editAclRequest(
     return checkAuth("editAclRequest.html", request, response, abstractAuthenticationToken);
   }
 
+  @RequestMapping(value = "/editSchemaRequest", method = RequestMethod.GET)
+  public String editSchemaRequest(
+      ModelMap model,
```

Comment:
is model required here? I don't see it used in this class

Reply:
As we added a new html page, we would need this.